### PR TITLE
events can have a different owner

### PR DESCRIPTION
**Pull request by alyxbb**
fun fact: myradio only makes me want to scream 99% of the time

### CODE REVIEW

**Comment by ashhhleyyy** [comment]
skill issue

File: src/Classes/ServiceAPI/MyRadio_Event.php [hunk 1]
```diff
@@ -277,15 +295,15 @@ public static function create($data = [])
             }
         }
 
-        $hostid = MyRadio_User::getCurrentOrSystemUser()->getID();
+        //$hostid = MyRadio_User::getCurrentOrSystemUser()->getID();
```
[the remark this answers is not in the document]
Comment (alyxbb) [reply src/Classes/ServiceAPI/MyRadio_Event.php]: oops, meant to delet it

**Comment by markspolakovs** [comment]
Also, I know that this is a student society and not a workplace, but could we please aim for a slightly higher level of professionalism when it comes to commit messages. The messages you write are read by other people, which may include (a) your future employers and (b) the people who wrote the code that "makes you scream" (in the case of the files this PR modifies, that would be me). There's more appropriate places to vent about the quality of the code you're maintaining than a public PR description.

File: src/Classes/ServiceAPI/MyRadio_Event.php [hunk 2]
```diff
@@ -135,6 +135,21 @@ public function setEndTime(int $endTime): void
...
+     * @param int
+     */
+    public function setHostId(int $hostId): void
+    {
+      self::$db->query('UPDATE public.events
+          SET hostid = $2
+          WHERE eventid = $1', [
+          $this->getID(),
+          $hostId
+      ]);
```
Comment (markspolakovs) [inline src/Classes/ServiceAPI/MyRadio_Event.php:148]: Indentation seems a bit off here

**Review by markspolakovs — APPROVED** [review]

File: src/Classes/ServiceAPI/MyRadio_Event.php (hunk 2 above)
Comment (alyxbb) [reply src/Classes/ServiceAPI/MyRadio_Event.php:148]: its the same as everywhere else

**Pull request merged**